tests/test_commands.py

```diff
@@ -169,6 +180,12 @@ def test_drop_existing_db(mock_rethink_db_drop):
     run_drop(args)
 
 
+def test_drop_non_existing_db(mock_rethink_db_drop_not_exist):
+    from bigchaindb.commands.bigchain import run_drop
+    args = Namespace(config=None, yes=True)
+    run_drop(args)
```

Comment:
This test is similar to `test_bigchain_run_init_when_db_exists` and `test_drop_existing_db`, but I don't like those two test much because they are missing `assert`ions.
Fixing those two tests is out of the scope of this PR, but do you have any suggestions on how to improve your new `test_drop_non_existing_db` test?